.github/workflows/package-release.yml

```diff
@@ -0,0 +1,42 @@
+name: Package release
+
+on:
+  pull_request:
+    types: 
+      - closed
+    branches: 
+      - main
+
+jobs:
+  package-release:
+    runs-on: ubuntu-latest
+    if: |
+      startsWith(github.head_ref, 'release-'')
+      && github.event.pull_request.merged
+      
+    steps: 
+      - name: Checkout out repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12.x'
```

Comment:
Should we be using node `16.x` here? It's LTS and [12 stopped getting active support in 2020](https://endoflife.date/nodejs).

Reply:
Just found https://github.com/trussworks/react-uswds/pull/1743 😄

Reply:
I added https://github.com/trussworks/react-uswds/issues/1749 so that we don't have to worry about merging one PR before the other and choosing which one to update the node version of our CI jobs in.

That way we can merge #1743 and this PR at any time, while also making sure to update the node version that this CI and the rest of our existing workflows use